README.md

```diff
@@ -19,17 +17,19 @@ However, it does not provide a SSH export as the DnD Magento connector bundle do
 
 | EnhancedConnectorBundle | Akeneo PIM Community Edition |
 |:-----------------------:|:----------------------------:|
+| v1.6.*                  | v1.6.*                       |
```

Comment:
We can discuss this point. Event if we cannot align another bundle, does it mean we should not do it for this one ?